tracer/src/Datadog.Trace/Iast/SensitiveData/EvidenceRedactor.cs

```diff
@@ -21,11 +21,13 @@ internal class EvidenceRedactor
     private TimeSpan _timeout;
     private Dictionary<string, ITokenizer> _tokenizers;
 
-    public EvidenceRedactor(string keysPattern, string valuesPattern, TimeSpan timeout, IDatadogLogger? logger = null)
+    public EvidenceRedactor(string keysPattern, string valuesPattern, TimeSpan timeout, int truncationMaxValueLength, IDatadogLogger? logger = null)
     {
         _timeout = timeout;
         _logger = logger;
 
+        TruncationUtils.Init(truncationMaxValueLength);
```

Comment:
FWIW, this is just _asking_ for testing issues 🙁 Shared static state is evil 😛 I would strongly suggest passing the truncation value into the `EvidenceConverter` constructor, and making the truncation method a pure method instead of having a shared static